code/modules/reagents/oldchem/reagents/drink/reagents_alcohol.dm

```diff
@@ -11,31 +11,29 @@
 	var/dizzy_adj = 3
 	var/slurr_adj = 3
 	var/confused_adj = 2
-	var/slur_start = 65			//amount absorbed after which mob starts slurring
-	var/brawl_start = 75		//amount absorbed after which mob switches to drunken brawling as a fighting style
-	var/confused_start = 130	//amount absorbed after which mob starts confusing directions
-	var/vomit_start = 180	//amount absorbed after which mob starts vomitting
-	var/blur_start = 260	//amount absorbed after which mob starts getting blurred vision
-	var/pass_out = 325	//amount absorbed after which mob starts passing out
+	var/slur_start = 20			//amount absorbed after which mob starts slurring
+	var/brawl_start = 25		//amount absorbed after which mob switches to drunken brawling as a fighting style
+	var/confused_start = 30	//amount absorbed after which mob starts confusing directions
+	var/vomit_start = 45 //amount absorbed after which mob starts vomitting
+	var/blur_start = 70	//amount absorbed after which mob starts getting blurred vision
+	var/pass_out = 90	//amount absorbed after which mob starts passing out
```

Comment:
I realise they can be adjusted, but it seems a bit too messy to adjust all six of them for every single type of alcohol. I still don't like the system of a unit of beer getting you to the same level of drunkenness as a unit of whiskey. Those two are different and people should drink them differently.

What people said in [this](http://nanotrasen.se/phpBB3/viewtopic.php?f=12&t=7422) thread was that the drunkeness depended on tha number of "drinks" someone had. The amount of the beverage per drink should depend on its type. Making vodka more powerful than cider seems more sensible to me.